dll/win32/riched20/txtsrv.c

```diff
@@ -481,7 +552,11 @@ DECLSPEC_HIDDEN HRESULT __thiscall fnTextSrv_TxGetCachedSize(ITextServices *ifac
 
 #ifdef __ASM_USE_THISCALL_WRAPPER
 
+#ifdef _MSC_VER
+#define func) (void *) __stdcall_ ## func
```

Comment:
`#define func)`: isn't this line broken?
Would something like
`#define STDCALL(func) ## func`
work, instead?